TokenSale.sol

```diff
@@ -72,7 +83,7 @@ contract TokenSale is TokenSaleInterface, Token {
     function refund() noEther {
         if (now > closingTime
             && !isFunded
-            && msg.sender.send(balances[msg.sender])) // execute refund
+            && msg.sender.send(balances[msg.sender])) // Execute refund
```

Comment:
for inline comments (so not comments that describe a function) it's really an overkill to have to capitalize. These are comments for the developers.